### PR TITLE
[Sql] Bump API version

### LINKED ISSUE #33734: Release Azure.ResourceManager.Sql
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/sql/resource-manager
Readme Tag: https://github.com/ostojicb/azure-rest-api-specs/blob/ob_midb_crosssubpitr/specification/sql/resource-manager/readme.md#package-composite-v5
No later than 2023-02-24

### PR DESCRIPTION
**Pull request by fengzhou-msft**
Resolve #33734
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md [hunk 1]
```diff
@@ -4,19 +4,33 @@
...
+- Added `filter` and `expand` parameters to `Get` methods of `RestorableDroppedDatabaseResource` and `SqlDatabaseResource`.
+- Added `RevalidateDatabaseEncryptionProtector` and `RevertDatabaseEncryptionProtector` methods to `SqlDatabaseResource`.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- Upgraded API version of Instance Failover Groups to `2022-05-01-preview`.
```
Comment (ArthurMa1978) [inline sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md:18]: Are we planed to release a beta sdk or stable?

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs [hunk 2]
```diff
@@ -1752,6 +1765,8 @@ public partial class ManagedInstanceResource : Azure.ResourceManager.ArmResource
         public virtual Azure.ResourceManager.Sql.ManagedServerSecurityAlertPolicyCollection GetManagedServerSecurityAlertPolicies() { throw null; }
         public virtual Azure.Response<Azure.ResourceManager.Sql.ManagedServerSecurityAlertPolicyResource> GetManagedServerSecurityAlertPolicy(Azure.ResourceManager.Sql.Models.SqlSecurityAlertPolicyName securityAlertPolicyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Sql.ManagedServerSecurityAlertPolicyResource>> GetManagedServerSecurityAlertPolicyAsync(Azure.ResourceManager.Sql.Models.SqlSecurityAlertPolicyName securityAlertPolicyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.Sql.Models.OutboundEnvironmentEndpoint> GetOutboundNetworkDependenciesByManagedInstance(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (ArthurMa1978) [inline sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs:1768]: Can `ByManagedInstance` be removed?

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs [hunk 3]
```diff
@@ -6719,6 +7016,12 @@ public partial class NetworkIsolationSettings
         public Azure.Core.ResourceIdentifier SqlServerResourceId { get { throw null; } set { } }
         public Azure.Core.ResourceIdentifier StorageAccountResourceId { get { throw null; } set { } }
     }
+    public partial class OutboundEnvironmentEndpoint
```
Comment (ArthurMa1978) [inline sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs:7019]: Is this a unique name  among all RPs?

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md (hunk 1 above)
Comment (fengzhou-msft) [reply sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md:18]: Stable version. The tag is still `package-composite-v5` with some preview API versions, which is allowed exception for Sql.

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs (hunk 3 above)
Comment (mcgallan) [reply sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs]: fixed

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs (hunk 2 above)
Comment (mcgallan) [reply sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs:1768]: fixed

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md (hunk 1 above)
Comment (mcgallan) [reply sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md:18]: fixed

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**